cmd/manager/main.go

```diff
@@ -69,13 +70,8 @@ func main() {
 		log.Fatal(err)
 	}
 
-	// Set proxy API availability
-	k8sInterface, err := kubernetes.NewForConfig(cfg)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	err = proxy.SetProxyAvailability(k8sInterface.Discovery())
+	// Set OpenShift config API availability
+	err = configv1.SetConfigAPIAvailability(cfg)
```

Comment:
Thinking out loud, this is the second OpenShift specific api we query for. Do we possibly want to combine the checks for this and ClusterOperator into a more thoughtful mechanism? This is outside the scope of this PR though.

Reply:
This is actually the third with the addition of the proxy api.

Reply:
I'd approve standardizing how we check for APIs.

Reply:
I agree with standardizing it and I am made the first inroad here while constraining it to the scope of the functionality being added. I did not want to try and rewrite all of it here.